bankly-app/models/user.js

```diff
@@ -79,16 +77,26 @@ class User {
    *
    * */
 
+  //BUG are parameters needed here?
   static async getAll(username, password) {
```

Comment:
May I inquire why the user's password is being passed as a parameter?

Reply:
@lorezi, these parameters were in the source code written by Springboard. I agree that they aren't needed. I'll remove them, ensure all tests are still passing, and merge.